source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -273,12 +280,12 @@ cf update-service custom-domains-production \
 
 #### Changing your custom domain's DNS records
 
-Once you have removed the custom domain(s) from your cdn-route service, you should delete any associated DNS records. This stops the removed domains' DNS records from pointing towards the GOV.UK PaaS. You should:
+Once you have removed the custom domain(s) from your cdn-route service, you should delete any associated DNS records. This stops the removed domains' DNS records from pointing towards GOV.UK PaaS. You should:
 
-- delete or amend the subdomain's CNAME so that it does not point to the GOV.UK PaaS
-- delete the subdomain's TXT record you created when you configured the subdomain to work with the GOV.UK PaaS
+- delete or amend the subdomain's CNAME so that it does not point to GOV.UK PaaS
+- delete the subdomain's TXT record you created when you configured the subdomain to work with GOV.UK PaaS
 
-<%= warning_text('Do not stop your domains` DNS records from pointing towards the GOV.UK PaaS before you have removed those custom domain(s) from your cdn-route service. <br><br> If you do, all of your domains` TLS certificates will fail to renew without warning at some point over the next 90 days. Your users may not be able to use your service.') %>
+<%= warning_text('Do not stop your domains` DNS records from pointing towards GOV.UK PaaS before you have removed those custom domain(s) from your cdn-route service. <br><br> If you do, all of your domains` TLS certificates will fail to renew without warning at some point over the next 90 days. Your users may not be able to use your service.') %>
```

Comment:
This is best practice but it doesn't clearly explain what is going on.
```suggestion
<%= warning_text('Do not stop your domains` DNS records from pointing towards GOV.UK PaaS before you have removed those custom domain(s) from your cdn-route service. <br><br> If you do, all of the domains` associated with that cdn-route service with fail to renew their TLS certificates without warning at some point over the next 90 days. Your users may not be able to use your service.') %>
```

Reply:
Making clearer what the safe thing to do is:
```suggestion
<%= warning_text('Remove a custom domain from your cdn-route service before removing the DNS records pointing at GOV.UK PaaS. <br><br>Otherwise all of the domains` associated with that cdn-route service will fail to renew their TLS certificates without warning at some point over the next 90 days. Your users may not be able to use your service.') %>
```